src/bun.js/ipc.zig

```diff
@@ -237,8 +326,8 @@ const SocketIPCData = struct {
 
     incoming: bun.ByteList = .{}, // Maybe we should use StreamBuffer here as well
     outgoing: bun.io.StreamBuffer = .{},
-
     has_written_version: if (Environment.allow_assert) u1 else u0 = 0,
+    iimh: node_cluster_binding.InternalMsgHolder = .{},
```

Comment:
can we use the full name instead of an acronym? who will know what an "iimh" is?